application/aam-backend-service/src/main/kotlin/com/aamdigital/aambackendservice/security/LocalDevelopmentConfiguration.kt

```diff
@@ -5,9 +5,14 @@ import org.springframework.boot.web.client.RestTemplateBuilder
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.annotation.Profile
+import org.springframework.http.client.SimpleClientHttpRequestFactory
 import org.springframework.security.oauth2.jwt.JwtDecoder
 import org.springframework.security.oauth2.jwt.NimbusJwtDecoder
+import org.springframework.web.client.RestClient
 import org.springframework.web.client.RestTemplate
+import java.net.HttpURLConnection
+import javax.net.ssl.HttpsURLConnection
+
 
 @Configuration
```

Comment:
Unrelated ... but I am not really sure what this config is about. Could you add a sentence as JavaDoc here about the purpose / context of "local-development" please? :blush:

Reply:
This Config will configure the self-signed ssl certificate from the cadd reverse proxy as trusted, when the local-development profile is active.